apps/backend/src/libs/modules/middleware/token-verification/token-verification.middleware.ts

```diff
@@ -0,0 +1,10 @@
+import { FastifyReply, FastifyRequest } from "fastify";
+
+const tokenVerificationMiddleware = async (
```

Comment:
No need to have a middleware for that. You receive a request -> take token -> decode token and get user id -> get user and send it to the client